src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_acr_agentpool_commands.py

```diff
@@ -3,11 +3,11 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from azure.cli.testsdk import ScenarioTest, StorageAccountPreparer, ResourceGroupPreparer, record_only
-
+from azure.cli.testsdk import ScenarioTest, StorageAccountPreparer, ResourceGroupPreparer, record_only, live_only
 
 class AcrAgentPoolCommandsTests(ScenarioTest):
 
+    @live_only()
```

Comment:
Can you please add a comment to explain why need to add `@live_only()` to this test?